### PR TITLE
Centered nav for artists / profiles

### PR DESCRIPTION
**Pull request by localastronaut**

### CODE REVIEW

File: static/css/perdiem.css [hunk 1]
```diff
@@ -276,3 +276,12 @@ form > input[type=submit] {
     display: block;
     margin: 0 auto;
 }
+
+ul.tabs {
+    text-align: center;
+}
+
+ul.tabs li {
+    float: none !important;
```
Comment (RevolutionTech) [inline static/css/perdiem.css]: I'm assuming you needed `!important` because of styles from Foundation?

**Comment by RevolutionTech** [comment]
Looks like this branch has conflicts (likely from the other PR I just merged in). Can you resolve these?

File: static/css/perdiem.css (hunk 1 above)
Comment (localastronaut) [inline static/css/perdiem.css]: Actually, it works without !important. I couldn't get it to work and saw that on a foundation forum and it worked, but seems to be working without it as well. Re-commiting

Comment (RevolutionTech) [inline static/css/perdiem.css]: Sweet!

**Pull request merged**